whylog/tests/tests_log_reader/file_reader.py

```diff
@@ -0,0 +1,86 @@
+import six
+
+
+class DataGeneratorLogSource(object):
+    def __init__(self, start_time, time_interval, number_of_lines, line_padding):
+        self._start_time = start_time
+        self._time_interval = time_interval
+        self._number_of_lines = number_of_lines
+        self._line_padding = line_padding
+        self._position = 0
+
+    def _deduce_line_no(self, offset):
+        return offset / self._line_padding
+
+    def _position_in_line(self, offset):
+        return offset % self._line_padding
+
+    def _get_line(self, line_no):
+        current_line_time = self._start_time + line_no * self._time_interval
+        current_line_time_str = str(current_line_time) + " "
```

Comment:
`str()` is too simple for some tests. There should be a format defined in the constructor, and here `current_line_time.strftime(self._format)` should be used.